src/librustc/mir/interpret/error.rs

```diff
@@ -247,6 +247,7 @@ pub enum InterpError<'tcx, O> {
     DanglingPointerDeref,
     DoubleFree,
     InvalidMemoryAccess,
+    FunctionPointerTyMismatch(FnSig<'tcx>, FnSig<'tcx>),
```

Comment:
this wasn't here before this PR, maybe a rebase gone wrong?

Reply:
This was also taken from [RFC](https://gist.github.com/TheDarkula/d5993ff5ec66338ad6063db044068f9f#file-error-rs-L6).

Is it not needed?